3rdparty/kdtoolbox/singleshot_connect/singleshot_connect.h

```diff
@@ -0,0 +1,120 @@
+/****************************************************************************
```

Comment:
This just adds a feature to enable single-shot Qt slot connections, just like there are single-shot timers.

Reply:
Maybe - but because of it's licence you MUST also add it to `dlgAboutDialog.cpp` - which should work out to be something like this:
```diff
     QString QtKeyChainHeader(tr("<h2><u>QtKeyChain - Platform-independent Qt API for storing passwords securely</u></h2>"
                                  "<h3>Copyright © 2011-2019 Frank Osterfeld &lt;frank.osterfeld@gmail.com&gt;.</h3>"));

+    QString SingleConnectHeader(tr("<h2><u>singleshot_connect.h - part of KDToolBox</u><br>(https://github.com/KDAB/KDToolBox).</h2>"
+                                   "<h3>Copyright © 2020-2021 Klarälvdalens Datakonsult AB, a KDAB Group company, &lt;info@kdab.comF&gt;.</h3>"));

// ... other stuff:

     license_3rdParty_texts.append(QStringLiteral("<hr><br>"
                                                 "%35%36")
                                  .arg(QtKeyChainHeader,                       // 35 - QtKeyChain header - translatable
                                       BSD2Clause_Body                         // 36 - QtKeyChain body BSD2 ("AUTHOR") - not translatable
                                       .arg(QLatin1String("AUTHOR"), QLatin1String("AUTHOR"))));

+    license_3rdParty_texts.append(QStringLiteral("<hr>%37%38")
+                                  .arg(SingleConnectHeader,              // 37 - singleshot_connect header - translatable
+                                       MIT_Body));                       // 38 - singleshot_connect body MIT - not translatable```

     license_3rdParty_texts.append(QStringLiteral("</body></html>"));